examples/fine-tuning/create_dataset.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+if [ "$#" -ne 2 ] || ! [ -d "$2" ]; then
+  echo "Usage: $0 <NEW_DATASET_DIR> <MNIST_DIR>" >&2
+  exit 1
+fi
+
+NEW_DATASET_DIR=$1
+MNIST_DIR=$2
+
+# create new dataset directories
+mkdir -p $NEW_DATASET_DIR/odd
+mkdir -p $NEW_DATASET_DIR/even
+
+# create symbolic links
+for digit in 0 2 4 6 8
+   do cp $MNIST_DIR/$digit/*.png $NEW_DATASET_DIR/even/
```

Comment:
If I create the dataset doing:
```
# create symbolic links
for digit in 0 2 4 6 8
   do ln -s $MNIST_DIR/$digit $NEW_DATASET_DIR/even/$digit
done
for digit in 1 3 5 7 9
   do ln -s $MNIST_DIR/$digit $NEW_DATASET_DIR/odd/$digit
done
```
Then the dataset tree has one sub-folder per digit and if I set the maximum number of samples per class to a small number I get only one type of digit per class (I just tried again and all even numbers of `8`s). Hence my suspicion that shuffling didn't work across folders but maybe this is only a problem in the presence of a "maximum samples per class"...